tests/snappi_tests/files/helper.py

```diff
@@ -151,3 +151,38 @@ def revert_config_and_reload(node, results=None):
     # parallel_run(revert_config_and_reload, {}, {}, list(args), timeout=900)
     for duthost in args:
         revert_config_and_reload(node=duthost)
+
+
+@pytest.fixture(autouse=True)
+def enable_debug_shell(setup_ports_and_dut):  # noqa: F811
+    _, _, snappi_ports = setup_ports_and_dut
+    rx_duthost = snappi_ports[0]['duthost']
+
+    if is_cisco_device(rx_duthost):
+        dutport = snappi_ports[0]['peer_port']
+        asic_namespace_string = ""
+        syncd_string = "syncd"
+        if rx_duthost.is_multi_asic:
+            asic = rx_duthost.get_port_asic_instance(dutport)
+            asic_namespace_string = " -n " + asic.namespace
+            asic_id = rx_duthost.get_asic_id_from_namespace(asic.namespace)
+            syncd_string += asic_id
```

Comment:
@sreejithsreekumaran : This should be:
syncd_string += str(asic_id)

Reply:
done